### PR TITLE
[Remove HCA - Part 1] Replace hca references with dbio

### PR DESCRIPTION
**Pull request by chmreid**
This replaces references to HCA with DataBiosphere, replaces references to `hca` CLI utility with `dbio` CLI utility, replaces references to `HCA_CONFIG_FILE` with references to `DBIO_CONFIG_FILE`

This PR changes instructions to use `dbio`, but there is still work to be done to actually switch over the CLI utility to be named `dbio`

Relates to #3 - completes some of the work required for it, but not all.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -65,13 +58,13 @@ for more information about service accounts. Use the `Google Cloud IAM web conso
 Development
 -----------
 To develop on the CLI, first run ``pip install -r requirements-dev.txt``. You can install your locally modified copy of 
-the hca package by running ``make install`` in the repository root directory.
+the ``dbio`` package by running ``make install`` in the repository root directory.
 
-To use the command line interface with a local or test DSS, first run ``hca`` (or ``scripts/hca`` if you want to use the
-package in-place from the repository root directory). This will create the file ``~/.config/hca/config.json``, which you
+To use the command line interface with a local or test DSS, first run ``dbio`` (or ``scripts/dbio`` if you want to use the
+package in-place from the repository root directory). This will create the file ``~/.config/dbio/config.json``, which you
```
Comment (amarjandu) [inline README.rst:63]: for the config file to be changed you have to modify https://github.com/DataBiosphere/data-store-cli/blob/c1e381f917c9b607a64b3243434470d578242559/hca/config.py#L12 the `name` here to dbio

File: README.rst [hunk 2]
```diff
@@ -1,62 +1,55 @@
...
+The DataBiosphere CLI supports ingesting configuration from a configurable array of sources. Each source is a JSON file.
 Configuration sources that follow the first source update the configuration using recursive dictionary merging. Sources
 are enumerated in the following order (i.e., in order of increasing priority):
 
-- Site-wide configuration source, ``/etc/hca/config.json``
-- User configuration source, ``~/.config/hca/config.json``
-- Any sources listed in the colon-delimited variable ``HCA_CONFIG_FILE``
+- Site-wide configuration source, ``/etc/dbio/config.json``
+- User configuration source, ``~/.config/dbio/config.json``
+- Data store specific configuration file, ``~/.config/dss/config.json``
```
Comment (amarjandu) [inline README.rst]: can omit this line, the dss config inside the user-config.

**Review by amarjandu — CHANGES_REQUESTED** [review]
just need to change the config.name other than that looks solid.



we also need testing CI for this.

**Comment by chmreid** [comment]
FYI I anticipate this PR will be merged in alongside a few other PRs that will make the necessary changes for these readme instructions to actually work...

File: README.rst (hunk 2 above)
Comment (chmreid) [reply README.rst]: Addressed by 2c25750

File: README.rst (hunk 1 above)
Comment (chmreid) [reply README.rst:63]: Will make this change in a later PR... Thanks for pointing out the location in the source code!

Comment (chmreid) [reply README.rst:63]: Addressed in #18

**Review by amarjandu — APPROVED** [review]

**Pull request merged**